site/source/tutorials/9001_dojo1_dijits/index.md

```diff
@@ -0,0 +1,187 @@
+
+The `DijitWrapper` function takes two arguments.  The first is the Dojo 1 `declare` class.  The second is an optional string, which represents that tag name that should be used when the Dojo 2 widgeting system needs to create a _DOM stub_ to attach the Dijit instance to.  It defaults to `div` if not supplied.
+
+{% task 'Wrap Dijits.' %}
+
+{% aside 'Information' %}
+All Dojo 1 modules are AMD modules without default exports.  Because of this, you have to import them using the `import * as Dijit from 'module/id';` syntax.
+{% endaside %}
+
+We recommend that any Dijits that you wrap, that you create them as individual modules in `/src/dijit` in the same path structure they are located in the source `dijit` package.
+
+{% instruction 'Create `/src/dijit/Fieldset.ts` with the following content:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/dijit/Fieldset.ts' %}
+
+{% instruction 'Create `/src/dijit/form/Button.ts` with the following content:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/dijit/form/Button.ts' %}
+
+{% instruction 'Create `/src/dijit/form/TextBox.ts` with the following content:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/dijit/form/TextBox.ts' %}
+
+{% task 'Importing Dijits.' %}
+
+Now that you have wrapped Dijits that you want to use, we need to import them into the Dojo 2 which will contain them.
+
+{% instruction 'Import Dijits into `/src/widgets/WorkerForm.ts`:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/WorkerForm.ts' lines:5-7 %}
+
+{% section %}
+
+## Placing Dijits
+
+{% task 'Adding to a widget\'s render function.' %}
+
+Now that we have imported our wrapped Dijits into the module we want to use them in, we need to add them to the render function.
+
+{% instruction 'Add imported Dijits to the `WorkerForm` widget\'s render function:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/WorkerForm.ts' lines:44-74 %}
+
+{% aside 'Information' %}
+The <em>properties</em> for a wrapped Dijit are equivalent to the <em>params</em> that can be passed when constructing a Dijit.  For more information on what parameters are possible on Dijits, you can refer to the [Dojo 1 API documentation](https://dojotoolkit.org/api/).
+{% endaside %}
+
+Once Dijits are wrapped, they behave very similar to Dojo 2 widgets.  Coupled with the `dojo-typings` TypeScript typings, you get code completion as well when specifying the _properties_ and TypeScript should help ensure the values of the properties are valid.
+
+The one restriction to be aware of is that any wrapped Dijit can only have other wrapped Dijit's as children.  This is because all Dojo 2 widget's DOM is represented virtually, but all Dojo 1 Dijits directly manage their own DOM.  Having virtual DOM children is not something a Dijit is capable of dealing with.
```

Comment:
Perhaps replace with: One restriction to keep in mind is that wrapped Dijits may only have other wrapped Dijit's as their children. Dojo 2 widget's represented their [DOM virtually]((../comingsoon.html)), while Dojo 2 Dijits directly manage their own DOM. As such, Dijits were not designed with the virtual DOM in mind, and cannot managed virtual DOM children. 

Coming soon will link to result of https://github.com/dojo/dojo.io/pull/180